src/containers/product-details/index.js

```diff
@@ -200,323 +204,341 @@ const ProductDetailsArea = memo(
       auctionNextPriceDrop &&
       auctionNextPriceDrop.scheduledTime > new Date().getTime();
 
-    const isActionsAvailable = () => {
```

Comment:
I wonder if some of this logic was why some of the boxes disappeared after a bid was made?